cloud/deploy/deploy.go

```diff
@@ -140,14 +141,16 @@ func deployDags(path, dagsPath, runtimeID string, client astro.Client) (string,
 	// Check the dags directory
 	monitoringDagPath := filepath.Join(dagsPath, "astronomer_monitoring_dag.py")
 
-	// Create monitoring dag file
-	err := fileutil.WriteStringToFile(monitoringDagPath, airflow.MonitoringDag)
-	if err != nil {
-		return "", err
+	if !organization.IsOrgHosted() {
```

Comment:
Does this cover both `HostedDedicated` and `HostedShared` deployments?

Reply:
I think so - as long as they are in a hosted org.

Reply:
can we also add check for deploymentType
if !organization.IsOrgHosted() && !(deployment.IsDeploymentDedicated(deploymentType) || deployment.IsDeploymentHosted(deploymentType) {

Reply:
Done in 6d49e15